.gitignore

```
@@ -1,3 +1,12 @@
+# Ignore files that examples create
+t10k-images-idx3-ubyte*
+t10k-labels-idx1-ubyte*
+train-images-idx3-ubyte*
+train-labels-idx1-ubyte*
+training.pt
+test.pt
+catboost_info
```

`catboost_info` seems to be a directory.

```suggestion
catboost_info/
```

---

You are right. Thank you!